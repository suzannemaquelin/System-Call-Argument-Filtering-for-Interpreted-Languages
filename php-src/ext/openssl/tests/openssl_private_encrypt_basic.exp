bool(true)

Warning: openssl_private_encrypt(): key param is not a valid private key in %s on line %d
bool(false)

Warning: openssl_private_encrypt(): key param is not a valid private key in %s on line %d
bool(false)

Warning: openssl_private_encrypt(): key param is not a valid private key in %s on line %d
bool(false)
bool(true)
string(4) "test"