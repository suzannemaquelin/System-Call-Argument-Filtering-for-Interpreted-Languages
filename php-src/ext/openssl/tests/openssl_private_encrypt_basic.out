Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
bool(true)

Warning: openssl_private_encrypt(): key param is not a valid private key in /root/php-src/ext/openssl/tests/openssl_private_encrypt_basic.php on line 15
bool(false)

Warning: openssl_private_encrypt(): key param is not a valid private key in /root/php-src/ext/openssl/tests/openssl_private_encrypt_basic.php on line 16
bool(false)

Warning: openssl_private_encrypt(): key param is not a valid private key in /root/php-src/ext/openssl/tests/openssl_private_encrypt_basic.php on line 17
bool(false)
bool(true)
string(4) "test"